client/my-sites/people/team-list/team.jsx

```diff
@@ -0,0 +1,152 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import classNames from 'classnames';
+import debugModule from 'debug';
+import omit from 'lodash/omit';
+import identity from 'lodash/identity';
+import { localize } from 'i18n-calypso';
+const debug = debugModule( 'calypso:my-sites:people:team-list' );
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import PeopleListItem from 'my-sites/people/people-list-item';
+import UsersActions from 'lib/users/actions';
+import InfiniteList from 'components/infinite-list';
+import deterministicStringify from 'lib/deterministic-stringify';
+import NoResults from 'my-sites/no-results';
+import analytics from 'lib/analytics';
+import PeopleListSectionHeader from 'my-sites/people/people-list-section-header';
+
+/**
+ * Module Variables
+ */
+class Team extends Component {
+	static propTypes = {
+		translate: PropTypes.func,
+	};
+
+	static defaultProps = {
+		translate: identity,
+	};
+
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			bulkEditing: false
+		};
+	}
```

Comment:
no constructor is needed for setting default state which does not depend on props

```js
class extends Component {
	state = {
		bulkEditing: false,
	};
}
```

also, I find the naming a bit confusing. is this supposed to be `allowBulkEditing` or `hasBulkEditing` or `bulkEditingIsEnabled` or something like that?